### PR TITLE
Stack functions first upload

### PR DESCRIPTION
**Pull request by SammyDW**
3 out of 11 stack groups are finished, rest is "coming soon".

### CODE REVIEW

File: dataminer-overview/DataMiner_Stack_Functions/Stack_Data_Management.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+> - [Supported system data storage architectures](xref:Supported_system_data_storage_architectures) in the DataMiner User Guide
+> - [About databases](xref:Databases_about) in the DataMiner User Guide
+
+## Audit trailing
+
+**Documenting the flow of transactions.**
+
+DataMiner offers detailed, uniform, and fully consolidated audit trailing, easy to browse and search via the DataMiner Cube client UI, and providing insight in all activities in the operational ecosystem.
+
+The administrator can see the changes made per user, device/element, which resources, which actions, etc.
```
Comment (MariekeGO) [inline dataminer-overview/DataMiner_Stack_Functions/Stack_Data_Management.md:51]: This sentence isn't clear. Do you mean this?
```suggestion
The administrator can see the changes made per user or per device/element, including which resources, which actions, etc.
```

Comment (SammyDW) [reply dataminer-overview/DataMiner_Stack_Functions/Stack_Data_Management.md:51]: Yes, can be updated as such. Also , the images have been updated.

File: dataminer-overview/DataMiner_Stack_Functions/Stack_Data_Connectivity.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Easily manage signal routes.
+
+> [!TIP]
+>
+> *More info coming soon. Stay tuned!*
+
+## Connectivity Modeling
+
+Provides accuracy of the phase and connectivity details of the network.
```
Comment (MariekeGO) [inline dataminer-overview/DataMiner_Stack_Functions/Stack_Data_Connectivity.md:21]: This is rather vague and hard to interpret. Could you rephrase it?

**Review by MariekeGO — APPROVED** [review]
